packages/proxy/src/providers/util.ts

```diff
@@ -39,7 +39,10 @@ async function convertMediaUrl({
     throw new Error("Failed to get content type of the media");
   }
   const baseContentType = contentType.split(";")[0].trim();
-  if (!allowedMediaTypes.includes(baseContentType)) {
+  if (
+    allowedMediaTypes !== null &&
+    !allowedMediaTypes.includes(baseContentType)
```

Comment:
#nit you can `if (allowedMediaTypes?.includes(baseContentType))`

Reply:
I think that would throw an exception if allowed media types is specified and includes the type (which we dont want)